support-frontend/assets/components/tabs/tabs.tsx

```diff
@@ -13,24 +15,27 @@ type PropTypes = {
 	tabsLabel: string;
 	tabElement: TabElement;
 	tabs: TabProps[];
-	onTabChange: (tabName: string) => void;
+	onTabChange: (tabId: PaperFulfilmentOptions) => void;
+	theme?: 'legacyTabs' | 'paperTabs';
 };
 
 function Tabs({
 	tabsLabel,
 	tabElement,
 	tabs,
 	onTabChange,
+	theme = 'legacyTabs',
```

Comment:
I wonder if we'll end up feeling the complexity of sharing the `Tabs` component between old and new? Very happy to leave it as is for now, but if it ends up being hard to support both or we end up feeling constrained I think it's okay to create a new `Tabs` component for the new layout and then delete the old one once this work is complete.